### ISSUE TITLE
Error when running smartnoise-synth MWE

### ISSUE THREAD

**Issue by neat-web** [opendp/smartnoise-sdk#491]
### Subject of the issue
A `TypeError` is raised when running the minimal working example (MWE) from [the documentation](https://docs.smartnoise.org/synth/index.html#getting-started).

### Environment
* Windows 10, Python 3.8
* `smartnoise-synth` installed via pip (version 0.3.0)

### Steps to reproduce
I installed the latest `smartnoise-synth` version and tried to run the MWE.

### Expected behaviour
The synthesizer works and a sample is printed to the console.

### Actual behaviour
A `TypeError` is raised and the synthesizer does not work. The traceback is shown below:
```python
Traceback (most recent call last):
  ...
  File "...\venv\lib\site-packages\snsynth\base.py", line 91, in fit_sample
    self.fit(
  File "...\venv\lib\site-packages\snsynth\mwem.py", line 313, in fit
    train_data = self._get_train_data(
  File "...\venv\lib\site-packages\snsynth\base.py", line 148, in _get_train_data
    self._transformer = TableTransformer.create(data, style=style,
  File "...\venv\lib\site-packages\snsynth\transform\table.py", line 190, in create
    return cls.from_pandas(data, style=style, nullable=nullable, categorical_columns=categorical_columns, ordinal_columns=ordinal_columns, continuous_columns=continuous_columns)
  File "...\venv\lib\site-packages\snsynth\transform\table.py", line 159, in from_pandas
    return cls.from_column_names(df.columns, style=style, nullable=nullable, categorical_columns=categorical_columns, ordinal_columns=ordinal_columns, continuous_columns=continuous_columns)
  File "...\venv\lib\site-packages\snsynth\transform\table.py", line 155, in from_column_names
    transformers = TypeMap.get_transformers(column_names, style=style, nullable=nullable, categorical_columns=categorical_columns, ordinal_columns=ordinal_columns, continuous_columns=continuous_columns)
TypeError: 'classmethod' object is not callable
```

### Possible solution
I noticed that the source file `type_map.py` distributed e.g. in the `smartnoise-synth-0.3.0.tar.gz` from [PyPi](https://pypi.org/project/smartnoise-synth/#files) contains an invalid `@classmethod` definition in line 23. Here an excerpt from the source file:
```python
class TypeMap:
    @classmethod
    
    @classmethod
    def get_transformers(cls, column_names, style='gan', *ignore, nullable=False, categorical_columns=[], ordinal_columns=[], continuous_columns=[])
```
After removing the invalid line the MWE runs successfully.

**Comment by joshua-oss** [comment]
Thanks for reporting this!  I'm not sure how that typo made it into the package despite being caught in CI and fixed in the main branch.  I've pushed an updated version, 0.3.0.1, where it should be fixed.

**Comment by neat-web** [comment]
Thanks, the issue is fixed in version 0.3.0.1.
